### PR TITLE
feat: Add new fonts to styleguide

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Part one of font theming story
Added new fonts to the styleguide: https://nidigitalsolutions.jira.com/browse/REPLAT-4649

I opted for naming fonts per magazine sections as they are all specific to certain magazine sections we have (culture, style and ST)

Web:
screenshot 2018-12-13 at 11 40 52

Android:
screenshot_1544701238

### CODE REVIEW

File: packages/styleguide/styleguide.showcase.js [hunk 1]
```diff
@@ -93,6 +93,34 @@ const fontFixture = select => {
         </Text>
         {fontDisplayer(fonts.supporting, phrase, styleguide.fontSizes)}
       </View>
+      <View style={styles.showoffFontsContainer}>
+        <Text style={styles.headline}>
+          Style Magazine (Century Gothic Bold)
+        </Text>
+        <Text>
+          Used for theme specific headlines, drop caps and pull-quotes in style
```
Comment (aronshamash) [inline packages/styleguide/styleguide.showcase.js]: ..`in the Style magazine`

File: packages/styleguide/styleguide.showcase.js [hunk 2]
```diff
@@ -93,6 +93,34 @@ const fontFixture = select => {
...
+          magazines
+        </Text>
+        {fontDisplayer(fonts.stylemagazine, phrase, styleguide.fontSizes)}
+      </View>
+      <View style={styles.showoffFontsContainer}>
+        <Text style={styles.headline}>Culture Magazine (Flama Bold)</Text>
+        <Text>
+          Used for theme specific headlines, drop caps and pull-quotes in
+          culture magazines
```
Comment (aronshamash) [inline packages/styleguide/styleguide.showcase.js]: `in the Culture magazine`

File: packages/styleguide/styleguide.showcase.js [hunk 3]
```diff
@@ -93,6 +93,34 @@ const fontFixture = select => {
...
+        </Text>
+        {fontDisplayer(fonts.culturemagazine, phrase, styleguide.fontSizes)}
+      </View>
+      <View style={styles.showoffFontsContainer}>
+        <Text style={styles.headline}>
+          Sunday Times Magazine (Tiempos Headline Bold)
+        </Text>
+        <Text>
+          Used for theme specific headlines, drop caps and pull-quotes in the
```
Comment (aronshamash) [inline packages/styleguide/styleguide.showcase.js]: .. `in The Sunday Times Magazine`

File: packages/styleguide/src/fonts/fonts.js [hunk 4]
```diff
@@ -2,9 +2,12 @@ const fonts = {
   body: "TimesDigitalW04",
   bodyRegular: "TimesDigitalW04-Regular",
   bodyRegularSmallCaps: "TimesDigitalW04-RegularSC",
+  culturemagazine: "Flama-Bold",
```
[the remark this answers is not in the document]
Comment (colincclark) [reply packages/styleguide/src/fonts/fonts.js]: camelCase please

[the remark this answers is not in the document]
Comment (tuncaulubilge) [reply packages/styleguide/src/fonts/fonts.js]: Yeah, not sure how I missed that, cheers

File: lib/fetch-fonts.js [hunk 5]
```diff
@@ -88,6 +88,36 @@ const fonts = [
       `${fontCdn}/GillSans/GillSansMTStd-Medium-f147e4bbf2.woff`,
       `${fontCdn}/GillSans/GillSansMTStd-Medium-45ad758029.ttf`
     ]
+  },
+  {
+    fontName: "CenturyGothic-Bold",
```
Comment (colincclark) [inline lib/fetch-fonts.js]: these will need to be added to the .storybook/preview-head.html file too

Comment (tuncaulubilge) [reply lib/fetch-fonts.js]: :+1: added

**Pull request merged**